functions.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+
+function pokemonCardContainer($allPokemon) {
+    $pokemonNames = "";
+    foreach ($allPokemon as $pokemon) {
+        $pokemonNames .= '<div class="card-stats">'
+            . "<p>" . $pokemon['name'] . "</p>".
```

Comment:
`name` is capitalised in the db. Change db or variable name to resolve

Reply:
(Y)

Reply:
Done